package.json

```diff
@@ -57,14 +57,14 @@
     "@11ty/eleventy-plugin-syntaxhighlight": "^4.1.0",
     "@babel/core": "^7.19.6",
     "@babel/preset-env": "^7.19.4",
+    "@netlify/plugin-a11y": "^1.0.0-beta.1",
```

Comment:
@mirisuzanne It looks like the build consistently fails with this newer version of the plugin because Pa11y times out (and we have no way of increasing the timeout). It also seems like this plugin isn't really under active development. 😞

https://app.netlify.com/sites/oddleventy/deploys/63583f8f8d55e60008d3026f

I think our options would be:

1. Revert to the older v0.0.12 (released in May 2020)
2. Try only checking a subset of files, instead of all HTML files in our (large) site

Reply:
oh hmm. Well, I think it would be fair to test a sample like the pages in our main nav - and that's giving us a decent overview of some list and detail pages? Could add to the list from there, if there are other things we want to test.

Reply:
@mirisuzanne Looks like that worked, and found a lot of issues! https://app.netlify.com/sites/oddleventy/deploys/63584bb55e143c0009419955

Reply:
Merging this; added a new card to address the warnings: https://trello.com/c/zGdKgfhj/198-address-a11y-warnings